### PR TITLE
Bugfix/delete readonly folders

### PR DESCRIPTION
**Pull request by mgallien**

### CODE REVIEW

File: src/libsync/propagatorjobs.cpp [hunk 1]
```diff
@@ -368,6 +379,8 @@ void PropagateLocalRename::start()
         };
 #endif
 
+        auto originalFolder = FileSystem::FilePermissionsRestore{existingFile, FileSystem::FolderPermissions::ReadWrite};
```
[the remark this answers is not in the document]
Comment (mgallien) [reply src/libsync/propagatorjobs.cpp:382]: this is the object temporarily modifying the permissions while we rename a file or folder and restoring them
constructor is making the folder read/write
destructor is making it read-only again

[the remark this answers is not in the document]
Comment (mgallien) [reply src/libsync/propagatorjobs.cpp:382]: that said, the variable name is really bad

[the remark this answers is not in the document]
Comment (mgallien) [reply src/libsync/propagatorjobs.cpp]: rename done
made the variable constant like the 2 other occurrences

**Review by claucambra — APPROVED** [review]

**Pull request merged**